docs/source/release/v6.9.0/Diffraction/Powder/New_features/36216.rst

```diff
@@ -0,0 +1 @@
+- By utilizing new methods in EventList :ref:`DiffractionFocussing <algm-DiffractionFocussing>` has improved performance when using EventWorkspaces and `PreverseEvents=False`.
```

Comment:
Try rst rather than md
```suggestion
- By utilizing new methods in EventList :ref:`DiffractionFocussing <algm-DiffractionFocussing>` has improved performance when using EventWorkspaces and ``PreverseEvents=False``.
```